README.md

```diff
@@ -75,6 +75,47 @@ The output contains the specified term and all its ancestors up to `owl:Thing`.
 
 You may also specify which predicates you would like to include with `-p <term>`/`--predicate <term>` or `-P <file>`/`--predicates <file>`, where the file contains a list of predicate CURIEs or labels. Otherwise, the output includes all predicates. Since this extracts a hierarchy, unless you include the `-n` flag, `rdfs:subClassOf` will always be included.
 
+### `gizmos.search`
+
+The `search` module returns a list of JSON objects for use with the tree browser search bar.
+
+Usage in the command line:
+```
+python3 -m gizmos.search [path-to-database] [search-text] > [output-json]
+```
+
+Usage in Python (with defaults for optional parameters):
+```python
+json = gizmos.search(    # returns: JSON string
+    database,            # string: path to database
+    search_text,         # string: text to search
+    label="rdfs:label",  # string: term ID for label
+    short_label=None,    # string: term ID for short label
+    synonyms=None,       # list: term IDs for synonyms
+    limit=30             # int: max results to return
+)
+```
+
+Each returned object has the following attributes:
+* `id`: The term ID
+* `label`: The term `rdfs:label` (or other property if specified)
+* `short_label`: The term's short label property value, if provided
+* `synonym`: The term's synonym property value, if provided
+* `property`: The term ID of the matched property (e.g., `rdfs:label`)
+* `order`: The order in which the term will appear in the search results, from shortest to longest match
+
+By default, the label will use the `rdfs:label` property. You can override this with `--label <term>`/`-L <term>`.
+
+A short label is only included if you include a property with `--short-label <term>`/`-S <term>`. The same goes for synonyms, which are only included if `--synonym`/`-s <term>` is specified. You may include more than one synonym property with multiple `-s` options. Synonyms are only shown in the search result if they match the search text, whereas a short label is always shown (if the term has one).
+
+When both short label and synonym(s) are provided and the matching term has both properties, the search result is shown as:
+
+> label - short label - synonym
```

Comment:
We want to follow DICE here and start with the short label: `short label - label - synonym`